.github/workflows/ci_workflows.yml

```diff
@@ -152,7 +152,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: initial_checks
     container:
-      image: quay.io/pypa/manylinux1_i686
+      image: quay.io/pypa/manylinux2014_i686
```

Comment:
Is this really 32-bit? Can't tell from the test header.

Reply:
Yes, i686 is 32-bit.